### DISCUSSION TITLE
Add custom file filters in the PR "Files changes" tab

### DISCUSSION THREAD

**Discussion by bdibon-cs** [community/community#63464] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hey!

I think it would be nice to be able to filter out custom file types. For example in TS projects you can filter out `.ts`, `.snap` and so on with the file filter dropdown, but you cannot filter out by `.spec.ts`.

Have a nice day,

**Comment by mirzacsquad** [comment]
Having the capability for regex filtering would be fantastic.

**Comment by EricBatlle** [comment]
Maybe this extension I made can help you: https://github.com/EricBatlle/Github-PR-File-Filters
